### PR TITLE
[ticket/13946] Increase the CSS margin around [code] and [list]

### PR DESCRIPTION
**Pull request by s9e**
PHPBB3-13946

### CODE REVIEW

**Comment by marc1706** [comment]
Is there a reason you closed the quote one?

**Comment by marc1706** [comment]
Before:
13946_before
After:
13946_after

File: phpBB/styles/prosilver/theme/content.css [hunk 1]
```diff
@@ -383,8 +383,7 @@ dl.faq dt {
 }
 
 .content ul, .content ol {
-	margin-bottom: 1em;
-	margin-left: 3em;
+	margin: 1em 0 1.2em 3em;
```
Comment (marc1706) [inline phpBB/styles/prosilver/theme/content.css]: I just checked and changing it to this creates the exact same margins as the codebox has in Chrome:
```margin: 0.8em 0 0.9em 3em;```

I'd prefer if the resulting margins would be the same.

Comment (s9e) [inline phpBB/styles/prosilver/theme/content.css]: I didn't notice they were different from each other. Then I assume that code/list have different font sizes? In that case,  would it make sense to use px or--even better--**rem**? Apparently [rem is supported on everything but IE 7](http://caniuse.com/#feat=rem).

I don't care either way, `0.8em 0 0.9em 3em` is fine by me.

Comment (marc1706) [inline phpBB/styles/prosilver/theme/content.css]: See related comment here: https://github.com/phpbb/phpbb/pull/3649#issuecomment-106742826

**Comment by callumacrae** [comment]
Yes, I was wondering where that margin went :tada:

**Pull request merged**